catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -248,7 +249,22 @@ public void setProcessingMechanism(String processingMechanism) {
      * @param attributeOverrides - a list of attributes to override
      */
     public void setAttributeOverrides(List<String> attributeOverrides) {
-        this.attributeOverrides = attributeOverrides;
+        Map<String, Serializable> attributeOverrideMap = new HashMap<>();
+        for (String s : attributeOverrides) {
+            String[] keyValue = s.split("=");
+
+            if (keyValue.length != 2) {
+                LOGGER.debug("Got the wrong number of arguments for an attribute override");
```

Comment:
I think this should be higher than debug, perhaps even refuse to create the route. A malformed override is definitely not going to do what an admin wants it to do, and they should be informed of that.